src/view/Window.cxx

```diff
@@ -57,8 +57,6 @@ void Window::setActivity(iActivityPtr &&activity)
   LOG(LOG_DEBUG) << "Setting up new activity";
   m_Activity->setup(*m_Activity);
   LOG(LOG_DEBUG) << "Drawing new activity";
-  m_Activity->draw(*m_Renderer);
```

Comment:
Why did you remove these lines?

Reply:
This is UI event, so redraw will be called after.